client/src/components/Task/PlayPauseButton/PlayPauseButton.js

```diff
@@ -0,0 +1,101 @@
+import "./PlayPauseButton.modules.scss";
+import React from "react";
+import IconButton from "@material-ui/core/IconButton";
+import PauseIcon from '@material-ui/icons/Pause';
+import PlayArrowIcon from '@material-ui/icons/PlayArrow';
+import Box from "@material-ui/core/Box";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import Fade from "@material-ui/core/Fade";
+
+export default function PlayPauseButton(props) {
+    const [started, setStarted] = React.useState(false);
+    const [progress, setProgress] = React.useState(0.0);
+    const [timerID, setTimerID] = React.useState(0);
+    const {maxTime} = props;
+
+    //Callback when play pause button is pressed
+    function playClick() {
+        setStarted((prev) => {
+            let newVal = !prev;
+
+            //If now stopped, clear interval
+            if(!newVal)
```

Comment:
personally think these double negatives might be confusing and the var name is not descriptive which caused us to require an extra comment here, will be more descriptive to put

```
currStartStatus = !prevStartStatus
if currStartStatus == false {
  xxx
}
```